WTWSMS/events/VIET_Jewish_Flavor_Events.txt

```diff
@@ -3,7 +3,7 @@
 
 #FLAVOR EVENTS
 
-#A bigoted courtier questions why we must have Jews in the court
+#A courtier questions why we must have Jews in the court
```

Comment:
Soapbox commentary _in the code_. 

It'd be like seeing something bashing Trump or celebrating "X pride" in the code. It's tacky.